src/usr/htmgt/HBconfig

```diff
@@ -1,6 +1,5 @@
 config HTMGT
     default n
```

Comment:
There does need to be a corresponding change within HTMGT or else this will break certain systems when you disable IPMI.  There is an IPMI sensor ID that is used but is not guaranteed to be correct for non-IPMI systems.

One of the openpower partners did come up with a bit-swizzle function that converts the HUID into a 16 bit value as a sensor ID and that seemed to work.

I don't expect @rlippert to come up with the "fix" here, but when the Hostboot team imports this, we need to make sure it is handled.